src/components/PrayerTimeWidget.vue

```diff
@@ -1,5 +1,52 @@
 <template>
   <div class="prayer-time-widget">
+    <div class="notification-controls">
+      <button 
+        v-if="!notificationsEnabled" 
+        @click="enableNotifications" 
+        class="notification-btn enable-btn"
+      >
+        🔔 Enable Prayer Notifications
```

Comment:
@itsiqbal shall we give user a choice to reset notification. ? 
example: prayer notification enabled. click to disable.